client/scripts/views/components/reaction_button.ts

```diff
@@ -18,51 +20,81 @@ export enum ReactionType {
   Dislike = 'dislike'
 }
 
+
+const getDisplayedReactorsForPopup = (vnode:  m.Vnode<{post: OffchainThread | AnyProposal | OffchainComment<any>,
+  type: ReactionType, displayAsLink?: boolean, tooltip?: boolean, large?: boolean},
+    m._NoLifecycle<m.Component<{post: OffchainThread | AnyProposal | OffchainComment<any>,
```

Comment:
What does `m._NoLifecycle` do?

Reply:
I've just created 2 types for the reaction button component.